MAUI/DataGrid/columns.md

```diff
@@ -112,6 +112,7 @@ The [DataGridAutoGeneratingColumnEventArgs](https://help.syncfusion.com/cr/maui/
 
 * **Column**: This property returns the created column which can be customized.
 * **Cancel**: This property cancels the column creation.
+* **PropertyType**: This property provides the type of underlying model property for which the column is created.

```

Comment:
This property specifies the type of the underlying model property for which the column is created.

Reply:
Updated